Adyen/UI/Views/SubmitButton.swift

```diff
@@ -124,21 +125,25 @@ public final class SubmitButton: UIControl {
     private func configureConstraints() {
         backgroundView.adyen.anchor(inside: self)
         
+        let height = heightAnchor.constraint(equalToConstant: 50.0)
```

Comment:
it is not used

Reply:
it is activated at line 129. but maybe it's better to do all the constraint activation at once to avoid confusion.